recipes/gct/meta.yaml

```diff
@@ -0,0 +1,93 @@
+{% set name = "gct" %}
+{% set version = "6.2.1550507116" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://repo.gridcf.org/gct6/sources/gct-{{ version }}.tar.gz
+  sha256: a34bba682a252c35f67cbf25873dd04a4acb60eafef6f99317e1386331fe737a
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage('gct', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - automake
+    - make
+    - pkg-config
+    - libtool
+  host:
+    - openssl =1.0
```

Comment:
I'm not 100% sure we had `run_exports` back in version `1.0.*`. If only 1.0 works I would try:

```
  host:
    - openssl 1.0.2r
  run:
    - openssl 1.0.2r
```

Note that the pin must be `x.x.x`, so I would no do `=1.0` b/c things may be broken.